bpf/lib/icmp6.h

```diff
@@ -257,7 +257,7 @@ static inline int __icmp6_send_time_exceeded(struct __sk_buff *skb, int nh_off)
 		sum = compute_icmp6_csum(data, 56, ipv6hdr);
 		payload_len = htons(56);
 		trimlen = 56 - ntohs(ipv6hdr->payload_len);
-		if (skb_change_tail(skb, skb->len + trimlen) < 0)
+		if (skb_change_tail(skb, skb->len + trimlen, 0) < 0)
```

Comment:
Can you adapt the probe as well to detect the helper availability properly:?
https://github.com/cilium/cilium/blob/master/bpf/probes/skb_change_tail.c